src/loaders/omeXML.js

```diff
@@ -39,7 +39,10 @@ export default class OMEXML {
   }
 
   getChannelNames() {
-    return this._Pixels.Channel.map(channel => channel['@_Name']);
+    const { Channel } = this._Pixels;
+    return Array.isArray(Channel)
+      ? Channel.map(channel => channel['@_Name'])
+      : [Channel['@_Name']];
```

Comment:
So if there is just one channel, `Channel` is an object not array? good to know.

Reply:
Yeah, I wonder if there is some sort of assay-specific stuff that could be inferred, but I didn't see anything.